Project/Project/Services/ActorAPIService.cs

```diff
@@ -35,18 +36,7 @@ await _httpClient.GetAsync(
 
         if (celebrities == null || !celebrities.Any())
         {
-            return new Actor
-            {
-                Name = actorName,
-                NetWorth = 0,
-                Gender = "Not found",
-                Nationality = "Not found",
-                Height = 0,
-                Birthday = new DateTime(1001, 1, 1),
-                IsAlive = false,
-                Occupations = "Not found",
-                Image = imageUrl
-            };
```

Comment:
Please add this again because this lets users at least get the image of a actor, even if there is no more information about him/her. So a user can at least get a face to a name. If you think the user should be allowed to do that, please explain why this would be bad, else add the code again :)

Reply:
If the celebrity is null, it means he/she has no occupation, that means it is unnecessary to show image

Reply:
Ok, please check if this can be handle somehow else. Problem is that not a few actors dont have information but a image, which could be interesting for the user. If you cant find another way its fine todo like you do right now, but if you find a way please use that one.